nntrainer/tensor/manager.cpp

```diff
@@ -626,4 +626,50 @@ void Manager::deinitializeTensors() {
   tensors_initialized = false;
 }
 
+/**
+ * @brief     Create weights with the given spec
+ *
+ */
+std::vector<Weight *>
+Manager::requestWeights(const GraphNode &node,
+                        std::vector<Weight::Spec> &weights_spec) {
```

Comment:
how about `const std::vector<Weight::Spec> &` ?

Reply:
this creates issue `reference to local variable` issue.

Reply:
Right

Reply:
i think the copy will be optimized by the compiler.